packages/components/src/form-token-field/types.ts

```diff
@@ -182,6 +182,12 @@ export interface FormTokenFieldProps
 	 * @default false
 	 */
 	__nextHasNoMarginBottom?: boolean;
+	/**
+	 * If true add any incompleteTokenValue as a new token.
+	 *
+	 * @default false
+	 */
+	__experimentalAddOnBlur?: boolean;
```

Comment:
Since it's been established that `__experimental` prop prefixes don't really make a prop experimental after it's merged in WP core, I wonder if we should just remove that prefix.

Also, since there is already a `tokenizeOnSpace` prop, should we call this prop `tokenizeOnBlur` ?

Reply:
good point, have switched to `tokenizeOnBlur`